contracts/TwabRewards.sol

```diff
@@ -168,20 +215,21 @@ contract TwabRewards is ITwabRewards {
         uint8 _currentNumberOfEpochs = _promotion.numberOfEpochs;
 
         require(
-            _numberOfEpochs < (type(uint8).max - _currentNumberOfEpochs),
+            _numberOfEpochs <= (type(uint8).max - _currentNumberOfEpochs),
             "TwabRewards/epochs-over-limit"
         );
 
         uint8 _extendedNumberOfEpochs = _currentNumberOfEpochs + _numberOfEpochs;
```

Comment:
No need for this variable- it's only used once

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/48/commits/ac29ad436f9460f7eef04eb838a2306387498b24